### DISCUSSION TITLE
New Tweepy Logo

### DISCUSSION THREAD

**Discussion by joshthecoder** [tweepy/tweepy#1404] [Ideas]
Today we just use the Twitter logo on our site, but going forward I would like to come up with our own logo.
If you have any ideas or want to share your attempt at a logo please post it here!

**Comment by yohannawaliya** [comment]
I suggest  Tweet logo put under Microscope's lens!

**Comment by GuanacoDevs** [comment]
Spoiler alert: I'm not a graphic designer
The favicon uses the color scheme of GitHub, has a round white background will not be noticed here, I guess...
Favicon:
![favicon](https://user-images.githubusercontent.com/32036535/87276015-aa571f00-c49c-11ea-83f5-4525eaf55d62.png)
[svg](https://drive.google.com/file/d/1PA65ijzuhX8WHCIDkVpfbTIk7KmEdxN2/view?usp=sharing)
Apple Touch Favicon:
![apple-touch-icon](https://user-images.githubusercontent.com/32036535/87276004-a3c8a780-c49c-11ea-98fd-71c02f8dc9b2.png)
[svg](https://drive.google.com/file/d/1sopC2iAvLdHP0UcfNs5390eXJQa3h_hO/view?usp=sharing)

Tweepy Logo:
![tweepy-logo](https://user-images.githubusercontent.com/32036535/87276097-dd011780-c49c-11ea-980f-6b27e617faad.png)

Sources:
[Python](https://www.python.org/community/logos/)
[Twitter](https://about.twitter.com/en_us/company/brand-resources.html)

**Comment by itsayushisaxena** [comment]
> I suggest Tweet logo put under Microscope's lens!

Hey @WaliyaYohannaJoseph  ,
                  It is a good idea because tweepy is used for in-depth analysis of a tweet(magnification of tweet) but don't you think it would not give enough importance to python.
**P.S.** I liked your idea :)
Since, Tweepy is python library for Twitter API. Therefore, what I think is logo must be somewhere connecting the dots between Twitter and Python.
![titelbild](https://user-images.githubusercontent.com/45483037/95815056-8aecbe00-0d39-11eb-95b8-ea2c1152da8a.png)

How is this one?

**Comment by clawdius** [comment]
I guess i'll give it a try

![tweepy](https://user-images.githubusercontent.com/61589143/97068916-041bc900-15f6-11eb-94ce-fbeef5d50f02.png)

**Comment by luni3359** [comment]
I think this one leaves a wrong impression, it's like someone beheaded the poor thing.

**Reply by turkishmaid** [reply]
The "eye" in the yellow area of @clawdius' image should go down to the lower edge, so that it matches better with the Python logo.

**Comment by clawdius** [comment]
> I think this one leaves a wrong impression, it's like someone beheaded the poor thing.

i was drunk when i did that 😂
